site/assets/apexcharts.js

```diff
@@ -0,0 +1,38 @@
+/*!
```

Comment:
Same comment here. Since this is an external library that's manually added to the site's assets, it would be nice to have a comment that says where the code comes from , which version it is (although that's already shown below), and how to update it (e.g. copy new version code from ... and paste it below this comment).

Reply:
Same comment here as well ;)